src/components/Homepage/HomepageMonthlyGoodInfo.vue

```diff
@@ -0,0 +1,155 @@
+<template>
```

Comment:
New component with new type. We should generalize this, so the icons can be modified via contentful.

Reply:
Yea we can do some moving around and renaming later to start isolating these"Content Block" type components.